tensorflow_io/bigquery/python/ops/bigquery_api.py

```diff
@@ -179,7 +179,13 @@ def read_rows(self, stream):
     return _BigQueryDataset(self._client_resource, self._selected_fields,
                             self._output_types, self._avro_schema, stream)
 
-  def parallel_read_rows(self, cycle_length=None, sloppy=False, block_length=1):
+  @deprecation.deprecated_args(
```

Comment:
why do you need this?
As far as I understand you added support for sloppy using tf.data.Options.experimental_deterministic below

Reply:
Just saw a comment on PR, I think it makes sense